packaging/linux/root/usr/share/buildkite-agent/lsb/buildkite-agent.sh

```diff
@@ -15,10 +15,11 @@
 user="buildkite-agent"
 cmd="/usr/bin/buildkite-agent start"
 
-name=`basename $0`
-pid_file="/var/run/$name.pid"
-log="/var/log/$name.log"
-stderr_log="/var/log/$name.err"
+name=$(basename "$(readlink -f "$0")")
```

Comment:
Using basename does allow you to run multiple instances with (a bit) less fuss:
cp /etc/rc.d/init.d/buildkite-agent  /etc/rc.d/init.d/buildkite-agent-2
chkconfig --add buildkite-agent-2
service buildkite-agent-2 start